ffcx/options.py

```diff
@@ -20,6 +20,7 @@
 logger = logging.getLogger("ffcx")
 
 FFCX_DEFAULT_OPTIONS = {
+    "cuda": (bool, False, "generate CUDA wrapped versions of tabulate tensor functions", None),
```

Comment:
Again let's be more specific here and say nvrtc.

Reply:
Agreed